docs/getting-started/setup.md

```diff
@@ -10,33 +10,44 @@ To run this tutorial you just need two things:
 - [Docker](https://www.docker.com/get-started), including docker-compose. If you use Windows, please read our note on [Docker for Windows](troubleshooting/docker-on-windows.md),
 - The [source code of the example](https://github.com/smart-data-lake/getting-started).
 
-## Build docker image
+## Build Spark docker image
 
 - Download the source code of the example either via git or by [downloading the zip](https://github.com/smart-data-lake/getting-started/archive/refs/heads/master.zip) and extracting it.
-- Open up a terminal and change to the folder with the source, you should see a file called Dockerfile.
-- Then run:
+- Open up a terminal and change to the folder with the source, you should see a file called Dockerfile. 
+- Then run (note: this might take some time, but it's only needed once):
 
 
-    docker build -t smart-data-lake/gs1 .
+    docker build -t sdl-spark .
 
 
-## Run docker image
+## Compile Java Classes
 
-Let's see Smart Data Lake in action!
-Run the following commands in the same terminal:
+Compile included Java classes. Note: this might take some time, but it's only needed at the beginning or if Java code has changed.
```

Comment:
```suggestion
Compile included Scala classes. Note: this might take some time, but it's only needed at the beginning or if Scala code has changed.
```